python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,103 @@
+import argparse, os, sys, time
+from modules.bank.query import query_balances
+from utils.bank import print_balance_deductions
+from utils.commands import exec_command
+from utils.keys import fetch_account_address
+from utils.txs import fetch_seq_no, signed_tx, unsigned_tx, write_json
+from utils.types import account_type, num_txs_type
+
+DAEMON = os.getenv('DAEMON')
+DENOM = os.getenv('DENOM')
+CHAINID = os.getenv('CHAINID')
+DAEMON_HOME = os.getenv('DAEMON_HOME')
+HOME = os.getenv('HOME')
```

Comment:
can we instead have a `read_env()` to simplify this. Something like,

`chainid, daemon, denom, daemon_home, home = read_env()`

This is just an example. Let's use sorted order to be consistent

Reply:
Oh just realised that we already have `parse_args()` below, can we use that here too? https://github.com/cosmos/qa-integration/pull/19/files#diff-8631cbdc926441820979dde2d2cf7816b51a4d6168f4bacad13a7a066c0adc26R20